types/lodash-fp/lodash-fp.utils.errors.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import _ from 'lodash/fp'
+
+import { SafeCurry2, SafeCurry3, SafeCurry4 } from '../../src/libs/type-utils/lodash-fp-types'
+
+
+// Curry2 ////////////////////////////////////////////////////////////////
+
+const fnWith2ArgsFn1 = (_x: number, _y: string): string => {
+  return 'something interesting'
+}
+const fnWith2ArgsFn2 = (_x: number[], _y: string) : string => {
+  return 'something interesting'
+}
+const badFn = (_arg1: number): string => {
+  return 'mischief'
+}
+
+const c2Fn1 = _.curry(fnWith2ArgsFn1) as SafeCurry2<typeof fnWith2ArgsFn1>
```

Comment:
Would it be easier to follow if names like these were spelled out, like `curriedFnWith2ArgsFn1`?

Reply:
fixed up the names